cmd/preflight/cmd/check_container.go

```diff
@@ -238,24 +239,34 @@ func checkContainerPositionalArgs(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func checkContainerPreRunE(cmd *cobra.Command, args []string) error {
+	certificationProjectID := viper.GetString("certification_project_id")
+	validatedCertificationProjectID, err := validateCertificationProjectID(certificationProjectID)
+	if err != nil {
+		return err
+	}
+	if validatedCertificationProjectID != certificationProjectID {
+		viper.Set("certification_project_id", validatedCertificationProjectID)
+	}
+	return nil
+}
+
```

Comment:
Extraneous?

Reply:
Might be vestigial from an earlier revision, given the age of this patch.

I'll check it out.